tools/nnicmd/launcher.py

```diff
@@ -36,6 +36,20 @@
 import random
 import string
 
+def print_log_content(config_file_name):
+    '''print log information'''
+    stdout_full_path = os.path.join(NNICTL_HOME_DIR, config_file_name, 'stdout')
```

Comment:
can you use a constant module to define stdout_full_path/stderr_full_path? 
I remember these two paths are also used elsewhere, so suggest to keep only on declaration.